### PR TITLE
Blog app: add devise

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
# Blog app: add devise
> ***In this feature update:***
- [x] Remove current_user method in ApplicationController, devise will provide us with one.
- [x] [Install and setup devise](https://github.com/heartcombo/devise#getting-started).
  - Can register a new user.
  - User logs in with a combination of email and password.
  - Hashed password should be stored in the database.
  - Ask for confirmation of email.
  - Can reset password.
- [x] Modify the views in devise for registration and login to match the wireframes from the Sneak Peek and your styling.

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```
Comment (mmsesay) [inline app/views/devise/sessions/new.html.erb]: - [ ] Please ensure that your login and registration pages match the wireframe. I will suggest you make some modifications here.  

Your Design | Wireframe Design |
:----------------:|:------------------------:|
a1|a2

**Review by mmsesay — CHANGES_REQUESTED** [review]
Hi @Chimwemwe-127001  & @Buyaki01, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Add a button for users to sign out of the application after logging in.
- [ ] Fixing the logic for a user to reset their password in the application. However, I got an error when I tried it. Please see test demo to understand the point and making here.

Test Demo |
:-------------:|
a1


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/devise/sessions/new.html.erb (hunk 1 above)
Comment (Chimwemwe-127001) [reply app/views/devise/sessions/new.html.erb:46]: https://user-images.githubusercontent.com/52143815/161935716-ebe3a5bb-3c30-4a99-a76c-e29a930766ea.mp4

Comment (Chimwemwe-127001) [reply app/views/devise/sessions/new.html.erb:46]: Fixed!!

**Comment by Chimwemwe-127001** [comment]
https://user-images.githubusercontent.com/52143815/161935917-d537116a-3397-4223-9373-9535a1393bac.mp4

>Fixed!!

**Review by dasileker — APPROVED** [review]
### **Hi @Chimwemwe-127001,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**